app/helpers/claims_helper.rb

```diff
@@ -39,7 +39,11 @@ def show_message_controls?(claim)
   end
 
   def messaging_permitted?(message)
-    (current_user_is_external_user? && !message.claim.redeterminable?) || message.claim_action.present?
+    return true if message.claim_action.present?
+    if current_user_is_external_user?
```

Comment:
hmm, if the [redeterminable? method](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/641d716be93a5b6c009b6764c11a712a2f9e24b6/app/models/claim/base_claim.rb#L379-L381) not working here then it should probably be simplified